websites/S/Stremio/presence.ts

```diff
@@ -0,0 +1,179 @@
+const presence = new Presence({
+		clientId: "969208766807547917"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLMediaElement,
+	timestamp: [number, number],
+	pauseCheck: boolean,
+	search: HTMLInputElement,
+	genreSort: string,
+	title: string;
```

Comment:
No

Reply:
Most of these variables don't get their value reassigned so use a global variable instead

Reply:
Only video doesnt get reassigned